chasm/component_field_option.go

```diff
@@ -0,0 +1,39 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
plz use 2025 for all new files :)

Reply:
It is `make copyright`. I need to change `LICENSE` in the root.